testData.js

```diff
@@ -11,6 +11,12 @@ const exists = require('util').promisify(fs.exists);
 
 const readFile = require('util').promisify(fs.readFile);
 
+const retry_amount = 3;

```

Comment:
maybe require it like this - `const sleep = require("util").promisify(setTimeout);`